src/goto-instrument/module_dependencies.txt

```diff
@@ -10,5 +10,6 @@ langapi # should go away
 linking
 pointer-analysis
 solvers
+synthesizer
```

Comment:
Is this needed?  Should it be `goto-instrument/synthesizer`?

Reply:
I thought it is needed for the linter check. But it seems that it doesn't fail the check after being removed.

Reply:
OK; thanks.